### DISCUSSION TITLE
Problem with the authentification

### DISCUSSION THREAD

**Discussion by sipe-daniel** [cypress-io/cypress-realworld-app#1185] [General] [answered]
After running **yarn install && yarn dev** I inserted a password **s3cret** with the username **Tavares_Barrows** and I had a response : Username or password is invalid

I use ubuntu 20.04

**Comment by robertguss** [comment, marked as answer]
@sipe-daniel I just pulled down the latest version of the Real World App and did a yarn install and I was able to login with that user and password. I am not sure exactly why you are not able to log in. Can you try logging in with a different user and see if that works? For example try `Katharina_Bernier` password is the same `s3cret`

It is possible that the `database.json` file included in this repo got corrupted or changed on your machine. So maybe try to clone down a fresh copy of the real world app in a new location on your machine and try again. Please let me know if you are successful or are still having issues.

**Reply by sipe-daniel** [reply]
Thanks for your response It's working now, I cloned down a fresh copy of the real world app in a new location on my machine.
